jest/setup.js

```diff
@@ -105,6 +105,12 @@ const mockNativeModules = {
   FbRelayNativeAdapter: {
     updateCLC: jest.fn(),
   },
+  Geolocation: {
+    getCurrentPosition: jest.fn(),
+    watchPosition: jest.fn(),
+    clearWatch: jest.fn(),
```

Comment:
Yeah, you should only add a mock for the NativeModule, so you probably want to add startObserving there, and remove the `LocationObserver` mock